default.nix

```diff
@@ -39,11 +42,15 @@ let
     hs-pkgs = project;
     local-packages = local-packages;
   };
+  morley =
+    (pkgs.haskell-nix.hackage-package
+      { name = "morley"; version = "1.4.0"; compiler-nix-name = "ghc883"; }
```

Comment:
Btw, I wonder which optimization settings will be used when `morley` is built.

Reply:
AFAIU, cabal-to-nix uses `cabal v2-configure` which has `-O1` by default